index.bs

```diff
@@ -111,8 +111,9 @@ is left up to individual user-agent implementations.
 An [=/entry=] |a| is <dfn for="entry">the same as</dfn> an [=/entry=] |b| if |a| is equal to |b|, or
 if |a| and |b| are backed by the same file or directory on the local file system.
 
-Issue: TODO: Explain better how entries map to files on disk (multiple entries can map to the same file or
-directory on disk but an entry doesn't have to map to any file on disk).
+An [=/entry=] maps roughly to a path, but there is no guarantee this path exists on disk.
```

Comment:
@mkruisselbrink do you have a suggestion for a better way to word this?

I'm still hedging with the use of "roughly" here. I'm not sure how to toe the line in being as explicit as we can about what exactly the entry is, while allowing for entries to point to places other than disk (Drive, etc)